.github/workflow-templates/cargo-build/action.yml

```
@@ -27,7 +27,7 @@ runs:
       shell: bash
       run: |
         mkdir -p mold
-        wget https://github.com/rui314/mold/releases/download/v1.2.0/mold-1.2.0-$(uname -m)-linux.tar.gz -O - | tar -C $(realpath mold) --strip-components=1 -xzf -
+        wget https://github.com/rui314/mold/releases/download/v2.3.0/mold-2.3.0-$(uname -m)-linux.tar.gz -O - | tar -C $(realpath mold) --strip-components=1 -xzf -
```

```suggestion
        wget https://github.com/rui314/mold/releases/download/v2.30.0/mold-2.30.0-$(uname -m)-linux.tar.gz -O - | tar -C $(realpath mold) --strip-components=1 -xzf -
```

The latest mold version is 2.30.0. Also you have some conflicts with master.